### ISSUE TITLE
[PTDE] Tag 'wet' not found in `NAME_TAG_RE` for this shader subclass.

### ISSUE THREAD

**Issue by EclipsedButter** [Grimrukh/soulstruct-blender#74]
The DeS update seems to have introduced a bug where PTDE is expected to have a 'wet' shader, even though it already has 'Water'. Currently, I've just put in a key-value for 'wet' and copied the value 'Water' to it, which works as long as an import doesn't actually use it.

**Comment by Grimrukh** [comment]
Interesting - that bug would've already been there, so what that really implies is that for some reason, Soulstruct can't find the MTDBND for PTDE, and so is trying to guess known material properties from the name alone. Will investigate that (and also fix the tag).

**Comment by EclipsedButter** [comment]
Console output suggests it does find the MTDBND:
```
# INFO: Using MTDBND for pre-Elden Ring FLVERs.
# INFO: Found MTDBND 'Mtd.mtdbnd' in game directory: /SteamLibrary/steamapps/common/Dark Souls Prepare to Die Edition/DATA/mtd/Mtd.mtdbnd
```
Interestingly, it normally doesn't find the MATBINBND for ER. It's looking for a nonexistent directory `matbin`, doesn't find it, and then guesses. Despite your comments suggesting that ER was significantly worse for such guessing games, it doesn't have problems when it does that. I changed it so that it looked in the `material` directory instead, where they are actually found, and it still works fine. Additionally, while I don't have the old console output to prove it, if memory serves correctly (that's a big if), pre-2.1.0 PTDE was still finding the MTDBND, and not having this particular issue.

**Comment by Grimrukh** [comment]
Gotcha, very good to know. I'll check it out. Thanks for all your efforts on this repo!

**Comment by Grimrukh** [comment]
This should now be fixed in 2.1.6.
